### ISSUE TITLE
Failed to add endpoint with url containing "%(tenant_id)s"

### ISSUE THREAD

**Issue by suntech-lab** [terraform-provider-openstack/terraform-provider-openstack#1444]
# terraform version
Terraform v1.3.0
on linux_amd64
+ provider registry.terraform.io/terraform-provider-openstack/openstack v1.48.0

Failed to add endpoint with url containing "%".
For example, "http://10.250.20.200:8776/v3/%(tenant_id)s"

Error: URL is not valid: parse "http://10.250.20.200:8776/v3/%(tenant_id)s": invalid URL escape "%(t"

I know this is not a valid URL, but it is the "url" string in OpenStack endpoint.
I wonder if we can remove the validation against URL for endpoint?

**Comment by nikParasyr** [comment]
hello.

which resources does this concern?

**Comment by tonyliu0592** [comment]
The resource is openstack_identity_endpoint_v3.
Here is an example. It's not related to #1261.
```
resource "openstack_identity_service_v3" "swift" {                             
  name = "swift"
  type = "object-store"
}
resource "openstack_identity_endpoint_v3" "swift-admin" {                     
  name = "swift"
  interface = "admin"
  service_id = openstack_identity_service_v3.swift.id                         
  endpoint_region = openstack_identity_service_v3.swift.region                
  url = "http://10.250.52.201/swift/v1/AUTH_%(tenant_id)s"                    
}
```
When apply this, you will see the error message.

**Comment by kayrus** [comment]
Fixed in #1701
